app/controllers/spree/admin/payment_methods_controller.rb

```diff
@@ -126,6 +121,24 @@ def load_hubs
         # rubocop:enable Style/TernaryParentheses
       end
 
+      def load_providers
+        @providers = Gateway.providers.sort_by(&:name)
+
+        unless Rails.env.development? || Rails.env.test?
+          @providers.reject! { |provider| provider.name.include? "Bogus" }
```

Comment:
We've got so many branching conditionals scattered around that are just there to skip the "Bogus" payment gateways. I'm not sure they even have much value in the app, and especially not in development environment.

Anyway, just noting it. I don't think we need to address it or think about it here :ok_hand: